Design-Docs/IAM-federation/keystone-keycloak-federation.md

```diff
@@ -80,10 +86,33 @@ console and seems to be described in the standards too).
 
 ### Openstack CLI
 
-As demonstrated in a PoC in 2022-02, the Keystone container in the SCS testbed can
-be configured with the help of `mod_oauth2` to support a different kind of authentication
-flow, which allows openstack CLI to fetch an auth token directly from Keycloak in the first
-step and then pass that auth token to the Keystone container for authorization. This second
+The Keystone container in SCS is configured to support authentication flows
```

Comment:
Maybe better write:

"The Keystone container as deployed by OSISM, which builds on top of kolla-ansible and is part of the IaaS reference implementation of SCS, is configured

Reply:
Ok, I merged and pushed the changes.